src/memray/_memray/hooks.cpp

```diff
@@ -162,34 +164,6 @@ pymalloc_free(void* ctx, void* ptr) noexcept
     }
 }
 
-void*
-mmap(void* addr, size_t length, int prot, int flags, int fd, off_t offset) noexcept
```

Comment:
Some of the changes in this file appear unnecessary. Looks like you moved the `mmap` and `mmap64` and `munmap` functions around in the file, but they're not guarded by any additional macros or anything... This makes the diff harder to review than necessary

Reply:
Sorry, splitting this into commits was not easy :( I can try to restore the functions if you want

Reply:
No, as long as you didn't make any changes to the implementation of the functions, I'm happy. If you did make changes to the implementations, I didn't notice them through the diff noise.

Reply:
I did not, so I am leaving as this. Apologies for the noise.